src/layout/MainContent/MainContentStyles.ts

```diff
@@ -0,0 +1,9 @@
+import { styled } from "@mui/material/styles";
+import { Box } from "@mui/material";
+
+export const MainContentContainer = styled(Box)(({ theme }) => ({
```

Comment:
Use proper naming convention, use of same name? MainContentContainer - lead to confusion